tests/integration/utils.py

```diff
@@ -50,6 +50,7 @@ def create_new_request(self, payload):
         :rtype: Response
         :raises requests.exceptions.HTTPError: if the request to the Cachito API fails
         """
+        print(payload)
```

Comment:
This is intended, right? I would normally suggest to use logging for this instead, but I'd also be fine with adding a comment to explain that it is not just a forgotten debugging print :-)

Reply:
that's definitely not intended, thanks for noticing :)